tests/System.Buffers.Experimental.Tests/BufferFactory.cs

```diff
@@ -53,6 +53,14 @@ public static ReadOnlySequence<byte> Create(params byte[][] buffers)
             return Create(list.ToArray());
         }
 
+        public static ReadOnlySequence<byte> CreateUtf8(params string[] buffers)
+        {
+            if (buffers.Length == 1) return new ReadOnlySequence<byte>(Encoding.UTF8.GetBytes(buffers[0]));
+            var list = new List<Memory<byte>>();
+            foreach (var b in buffers) list.Add(Encoding.UTF8.GetBytes(b));
```

Comment:
nit: avoid using var here

also not: avoid single letter variable names

Reply:
Yeah, copying existing code around.

Reply:
Var is ok in this context btw